docs/en/latest/installation-guide.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Installation
+keywords:
+- apisix-java-plugin-runner
+- Installation
+  description: This document explains how to installation and use apisix-java-plugin-runner.
+---
+
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Overview
+
+This document explains how to install apisix-java-plugin-runner.
+
+Prerequisites
+-------------
+
+* JDK 11
+* APISIX 2.15.0
+* Refer to [Debug](how-it-works.md#debug)  to build the debug environment.
+
+Install
+-------
+
+1. Create a simple web application with Spring Boot, and choose Maven as the build tool.
+
+2. Add the GAV of `apisix-java-plugin-runner` in `pom.xml`, like:
+
+```xml
+<dependency>
+    <groupId>org.apache.apisix</groupId> 
+    <artifactId>apisix-runner-starter</artifactId>
+    <version>${project.version}</version>
```

Comment:
`${project.version}` is the project POM's version, _i.e._, if my POM is `0.1-SNAPSHOT`, it will use this value. It seems highly suspect: I'd suggest to replace it with `0.3.0`. Even better, we could update the value to the current version at documentation's build time.

Reply:
done